packages/blockchain-link/src/workers/solana/fee.ts

```diff
@@ -0,0 +1,37 @@
+import { Connection, Message, PublicKey } from '@solana/web3.js';
+import BigNumber from 'bignumber.js';
+
+const DEFAULT_COMPUTE_UNIT_PRICE = 100_000; // micro-lamports, value taken from other wallets
+const DEFAULT_COMPUTE_UNIT_LIMIT = 50_000; // sending tokens with token account creation requires ~28K units
+
+export const getBaseFee = async (api: Connection, message: string) => {
+    const result = await api.getFeeForMessage(Message.from(Buffer.from(message, 'hex')));
+    // The result can be null, for example if the transaction blockhash is invalid.
+    // In this case, we should fall back to the default fee.
+    if (result.value == null) {
+        throw new Error('Could not estimate fee for transaction.');
+    }
+
+    return result.value;
+};
+
+export const getPriorityFee = async (api: Connection, affectedAccounts: PublicKey[]) => {
+    const recentFees = await api.getRecentPrioritizationFees({
+        lockedWritableAccounts: affectedAccounts.map(a => new PublicKey(a)),
+    });
+
+    const networkPriorityFee = recentFees.map(a => a.prioritizationFee).sort((a, b) => b - a)[
+        Math.floor(recentFees.length / 4)
+    ]; // 25th percentile because many 0 priority fees are expected
+
+    const computeUnitPrice = new BigNumber(
+        Math.max(networkPriorityFee, DEFAULT_COMPUTE_UNIT_PRICE),
+    );
```

Comment:
Don't we want to `Math.min` here to save the user fees if the network isn't congested? 
i.e. if all the recent priority fees are 0, as is often the case on devnet, I'd expect to not have to pay any priority fees.

Reply:
No, we want to always set a priority fee just to be sure the transactions go through. Same is true for other wallets as well.